ui/npm-shrinkwrap.json

```diff
@@ -0,0 +1,156 @@
+{
+  "name": "cockroach-ui",
+  "version": "0.1.0",
+  "dependencies": {
```

Comment:
I tried this with dev dependencies but shrinkwrap wasn't working correctly.  We're not planning to publish the package, so normal dependancies will be fine.